app/lookups/land/path.rb

```diff
@@ -4,7 +4,7 @@ class Path < ApplicationRecord
 
     lookup_by :path, cache: 50, find_or_create: true
 
-    has_many :page_views
+    has_many :pageviews
```

Comment:
example of correctly referenced: https://github.com/Beyond-Finance/land/blob/beyond-main/app/lookups/land/http_method.rb